src/api/lib.py

```diff
@@ -0,0 +1,18 @@
+import os
+from uuid import uuid4
+
+
+def _make_db_uri(async_url: bool = False) -> str:
+    """Correctly make the database URi."""
+    user = os.environ["PG_USER"]
+    password = os.environ["PG_PASS"]
+    prefix = "postgresql"
+    if async_url:
+        prefix = f"{prefix}+asyncpg"
+
+    return f"{prefix}://{user}:{password}@localhost:5432/demos"
```

Comment:
If this function is written in src/api/app.py, then it should be deleted yeah? Or imported into app.py and the other function should be deleted.